modules/adding-node-iso-configs.adoc

```diff
@@ -68,6 +68,17 @@ For more information, see "Root device hints" in the "Setting up the environment
 The configuration must match the Host Network Management API defined in the link:https://nmstate.io/[nmstate documentation].
 |A dictionary of host network configuration objects.
 
+|cpuArchitecture
```

Comment:
Is cpuArchitecture nested within another spec? Jw I notice others in the table are nested. Same question for the other one.

Reply:
I don't believe so, it seems to be its own level per the actual code: https://github.com/openshift/installer/blob/cc5c347ed7a79d643f2859ec11a3de7fd26f3c53/pkg/asset/agent/joiner/addnodesconfig.go#L39-L41